Packages/com.unity.multiplayer.samples.coop/Utilities/Net/SessionManager.cs

```diff
@@ -109,9 +109,9 @@ private void Clear()
         /// <param name="clientGUID">This is the clientGUID that is unique to this client and persists accross multiple logins from the same client</param>
         /// <param name="sessionPlayerData">The player's initial data</param>
         /// <returns></returns>
-        public ConnectStatus OnClientApprovalCheck(ulong clientId, string clientGUID, T sessionPlayerData)
+        public bool OnClientApprovalCheck(ulong clientId, string clientGUID, T sessionPlayerData)
         {
-            ConnectStatus gameReturnStatus = ConnectStatus.Success;
+            bool success = true;
```

Comment:
It feels like this method is making less sense now :(
Let's go back to what's the intent here, why do we have this method: to check whether session manager has duplicates right? Why not just rename this to "HasDuplicates" or something along those lines. Then I would agree returning a bool makes sense

Reply:
It does state changes too, so "HasDuplicates" might not be the right name. But you get my drift

Reply:
much better